archeryutils/handicaps/handicap_equations.py

```diff
@@ -244,7 +248,9 @@ def arrow_score(target, h, hc_sys, hc_dat, arw_d=None):
 
     if target.scoring_system == "5_zone":
         s_bar = (
-            9 - 2 * sum(
+            9
+            - 2
+            * sum(
```

Comment:
I am willing to argue with the Black formatting here. I think everything on a single line is less readable. Is there a good argument for changing it as it suggests?

Reply:
It's an issue I've seen a lot with scientific code, since any serious equations can very easily run over the 88 characters max line length. I think the reasoning behind black here is that it's more consistent: each line is a self-contained expression, and operators are used as the line breaks. I agree that it looks awful though... I just left it the way black says to do it, since otherwise I'd have to go back and manually fix it every time I wanted to check my formatting. I think it might be possible to generalise the equations in some way so that we don't need the `if...elif...else` block, and we'll be able to write them cleaner in whatever new form they take. Something like:

```python
s_bar = target.max_score
# handle innermost ring
s_bar -= np.exp(-((target.inner_diameter + arw_rad) / sig_r) ** 2)
# handle other rings
s_bar -= target.score_increment * (
    np.sum(
        np.exp(
            -(((np.arange(2, tar_rings) * tar_dia / tar_zones) + arw_rad)  / sig_r) ** 2
        )
    )
)
# handle outermost ring
#...
# handle missing rings
# etc... probably got an unmatched parenthesis in there
```
I think this would be a better approach overall, as otherwise the handicap system needs to know how to interpret "10 zone 5 ring compound" instead of just using a generic Target. Having a large collection of similar equations is also quite error prone compared to having a single generic method.

Reply:
OK, yes, I like this idea.
Requires some sitting down with a pen and paper for a while...
But could be made general based on intrinsic properties of Target.
Not something to be done on this commit, however, so I'll open an issue.
And fine - given it doesn't need to be looked at too often I am happy to go with the black. ;P